### PR TITLE
sql server update

### PR DESCRIPTION
**Pull request by l0k0ms**
https://trello.com/c/9Fb0slho/

### CODE REVIEW

File: content/integrations/sqlserver.md [hunk 1]
```diff
@@ -6,14 +6,62 @@ kind: integration
...
 ## Overview
 
 Connect SQL Server to Datadog in order to:
 
   * Visualize your database performance.
   * Correlate the performance of SQL Server with the rest of your applications.
 
-{{< insert-example-links conf="sqlserver" check="sqlserver" >}}
+<div class="alert alert-warning">
+SQL Server check can only be run from a Windows environment
```
Comment (kmshultz) [inline content/integrations/sqlserver.md]: This belongs in the Compatibility section.

File: content/integrations/sqlserver.md [hunk 2]
```diff
@@ -6,14 +6,62 @@ kind: integration
...
+</div>
+
+## Configuration
+This integration requires a Datadog Agent version >= 3.2.3
```
Comment (kmshultz) [inline content/integrations/sqlserver.md]: This version is now so old, I'm not sure we need to point this out.

File: content/integrations/sqlserver.md [hunk 3]
```diff
@@ -6,14 +6,62 @@ kind: integration
...
+
+1. Make sure that your SQL Server instance supports SQL Server authentication by enabling "SQL Server and Windows Authentication mode" in the server properties.
```
Comment (kmshultz) [inline content/integrations/sqlserver.md]: If you look in integrations-core (e.g. for the MySQL check), I've been doing two sub-headers for configuration instructions like this (i.e. integrations where you first need to prepare the collected-from thing before connecting the Agent up):

### Prepare SQL Server

Make Sure your SQL Server instance supports...

Then create a read-only user for the Datadog Agent to use to collect metrics...

### Connect the Agent

Create a file `sqlserver.yaml` in the Agent's conf.d directory:

```
...
```

Restart the Agent to start sending SQL Server metrics to Datadog.

File: content/integrations/sqlserver.md (hunk 2 above)
Comment (l0k0ms) [inline content/integrations/sqlserver.md]: Some people are still using it. :/

File: content/integrations/sqlserver.md (hunk 3 above)
Comment (l0k0ms) [inline content/integrations/sqlserver.md]: I will add a PR about it.

File: content/integrations/sqlserver.md (hunk 1 above)
Comment (l0k0ms) [inline content/integrations/sqlserver.md]: 👍

File: content/integrations/sqlserver.md [hunk 4]
```diff
@@ -6,14 +6,64 @@ kind: integration
...
+
+  sqlserver
+  ---------
+      - instance #0 [OK]
+      - Collected 8 metrics & 0 events
+{{< /highlight >}}
+
+## Compatibility 
+
+<div class="alert alert-warning">
```
Comment (kmshultz) [inline content/integrations/sqlserver.md]: We're going way overboard (IMO) with the warn styling. I don't think we need it here.

Comment (l0k0ms) [inline content/integrations/sqlserver.md]: 👍 I will use it less from now on. :)

**Pull request merged**